src/EditorFeatures/Core/ReferenceHighlighting/NagivateToHighlightReferenceCommandHandler.cs

```diff
@@ -33,37 +37,58 @@ internal partial class NavigateToHighlightReferenceCommandHandler :
             _tagAggregatorFactory = tagAggregatorFactory ?? throw new ArgumentNullException(nameof(tagAggregatorFactory));
         }
 
-        public CommandState GetCommandState(NavigateToHighlightedReferenceCommandArgs args, Func<CommandState> nextHandler)
+        public CommandState GetCommandState(NavigateToNextHighlightedReferenceCommandArgs args)
+        {
+            return GetCommandStateWorker(args);
+        }
+
+        public CommandState GetCommandState(NavigateToPreviousHighlightedReferenceCommandArgs args)
+        {
+            return GetCommandStateWorker(args);
+        }
+
+        private CommandState GetCommandStateWorker(Microsoft.VisualStudio.Text.UI.Commanding.Commands.CommandArgs args)
```

Comment:
:bulb: `Microsoft.CodeAnalysis.Editor.CommandArgs` is no longer required. It can be removed, allowing clean-up for these fully qualified types.